Adyen/Constants/ClientConfig.cs

```diff
@@ -59,6 +59,9 @@ public class ClientConfig
         public static string LegalEntityManagementEndpointTest = "https://kyc-test.adyen.com/lem/";
         public static string LegalEntityManagementEndpointLive = "https://kyc-live.adyen.com/lem/";
         public static string LegalEntityManagementVersion = "v2";
+        public static string BalancePlatformEndpointTest = "https://balanceplatform-api-test.adyen.com/bcl/";
+        public static string BalancePlatformEnndpointLive = "https://balanceplatform-api-test.adyen.com/bcl/";
```

Comment:
Suggestion: rename to `BalancePlatformEnndpointLive` to `BalancePlatformEndpointLive` and point url to '-live' 👍

Reply:
wow stupid mistake my bad